src/__tests__/__snapshots__/cliEntry.test.js.snap

```diff
@@ -6,12 +6,10 @@ Starts a new webpack server
 
 Options:
 
-  --port [number]                 Port to run your webpack server, 8081 by default
-  --dev [true|false]              Whether to build in development mode, true by default
-  --minify [true|false]           Whether to minify the bundle, 'true' by default when dev=false
-  --platform <ios|android|all>    Platform to bundle for
```

Comment:
We still support this flag for bundling, right? Maybe this should be left here?

Reply:
This is `start` command snapshot not `bundle`